openmdao/recorders/case.py

```diff
@@ -49,28 +75,96 @@ def __init__(self, filename, counter, iteration_coordinate, timestamp, success,
         self.timestamp = timestamp
         self.success = success
         self.msg = msg
+        self.inputs = None
+        self.outputs = None
+        self.residuals = None
+        self.meta = meta
+        self.prom2abs = prom2abs
+        self.abs2prom = abs2prom
+
+        if inputs is not None and inputs.dtype.names:
+            self.inputs = PromotedToAbsoluteMap(inputs[0], prom2abs, abs2prom, False)
```

Comment:
nitpick, but should probably include the keyword in keyword arguments: `output=False`